pyproject.toml

```
@@ -5,7 +5,7 @@ authors = [{ name = "Ionworks Technologies Inc", email = "info@ionworks.com" }]
 readme = { file = "README.md", content-type = "text/markdown" }
 license = { file = "LICENSE.md" }
 requires-python = ">=3.10"
-version = "0.3.4"
+version = "0.3.7"
```

This will avoid the errors again until I have time to switch it over to correct versioning in the main branch
```suggestion
version = "0.0.0"
```